packages/react/src/contexts/ClerkContextProvider.tsx

```diff
@@ -49,8 +49,9 @@ export function ClerkContextProvider(props: ClerkContextProvider): JSX.Element |
 
   const { sessionId, session, userId, user, organization } = derivedState;
   const actor = session?.actor || null;
-  const orgId = organization?.id;
-  const orgRole = session?.user.organizationMemberships.find(m => m.organization.id === orgId)?.role;
+  const orgId = organization === null ? null : organization?.id;
+  const orgRole =
+    organization === null ? null : session?.user.organizationMemberships.find(m => m.organization.id === orgId)?.role;
```

Comment:
🔧 Let's make this drier and find the correct `organizationMembership` once before destructuring orgId and orgRole

Reply:
I don't think it looks any better since we have to check for null everywhere, but done!